packages/contracts-bedrock/.storage-layout

```diff
@@ -67,28 +67,44 @@
 +-------------------+----------------------------------------------------------+------+--------+-------+
 | Name              | Type                                                     | Slot | Offset | Bytes |
 +======================================================================================================+
-| _owner            | address                                                  | 0    | 0      | 20    |
+| _initialized      | uint8                                                    | 0    | 0      | 1     |
 |-------------------+----------------------------------------------------------+------+--------+-------|
-| sequencer         | address                                                  | 1    | 0      | 20    |
+| _initializing     | bool                                                     | 0    | 1      | 1     |
 |-------------------+----------------------------------------------------------+------+--------+-------|
-| latestBlockNumber | uint256                                                  | 2    | 0      | 32    |
+| __gap             | uint256[50]                                              | 1    | 0      | 1600  |
```

Comment:
Why is this gas useful here? In case the ownable implementation upgrades?

Reply:
Yeah. It's built into [OwnableUpgradeable](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/f5819bc47bdf0c049f9b1c2eed6b940cbff7ada2/contracts/access/OwnableUpgradeable.sol#L94)